bindings/c/test/mako/mako.cpp

```diff
@@ -1024,6 +1088,7 @@ int parseArguments(int argc, char* argv[], Arguments& args) {
 			{ "iteration", required_argument, NULL, 'i' },
 			{ "keylen", required_argument, NULL, ARG_KEYLEN },
 			{ "vallen", required_argument, NULL, ARG_VALLEN },
+			{ "tenants", required_argument, NULL, ARG_TENANTS },
```

Comment:
This may be something that should be defaulted to 0. That way old invocations do the same thing that they used to.